docs: Added the flame_svg section (#2059)
flame-engine/flame: doc/flame_svg/svg.md

<!-- markdownlint-disable MD013 -->
<p align="center">
  <a href="https://flame-engine.org">
    <img alt="flame" width="200px" src="https://user-images.githubusercontent.com/6718144/101553774-3bc7b000-39ad-11eb-8a6a-de2daa31bd64.png">
  </a>
</p>

> <!-- markdownlint-enable MD013 -->

Reviewer: This is not needed.

> # Flame SVG

Reviewer 2: This should explain how to use the `SvgComponent`

Author: Where can I get that info?

Reviewer 2: You can have a look at the code in the example code [here](https://examples.flame-engine.org/#/Svg_Svg%20Component), press `< >` in the upper right corner to get to the code.

Also this section: https://docs.flame-engine.org/1.4.0/flame/rendering/images.html#svg
should be moved to be under this package too.

# Flame SVG

Flame provides a simple API to render SVG images in your game.

> Svg support is provided by the `flame_svg` external package, be sure to put it in your pubspec file

Reviewer 2: ```suggestion
Svg support is provided by the `flame_svg` external package, be sure to add it to your pubspec file
```

to use it.

To use it just import the `Svg` class from `'package:flame_svg/flame_svg.dart'`, and use the
following snippet to render it on the canvas:

```dart
Svg svgInstance = Svg('android.svg');

final position = Vector2(100, 100);
final size = Vector2(300, 300);

svgInstance.renderPosition(canvas, position, size);
```

or use the [SvgComponent]:

```dart
class MyGame extends FlameGame {
    Future<void> onLoad() async {
      final svgInstance = await Svg.load('android.svg');
      final size = Vector2.all(100);
      final position = Vector2.all(100);
      final svgComponent = SvgComponent.fromSvg(
        size, 
        position, 
        svgInstance,

> );

Reviewer 2: Needs formatting, run this through `flutter format`

Author: Thx, gonna fix all asap :)

Author: Formatted!

> 

Reviewer 2: Whitespace that shouldn't be added

# Flame SVG

Flame provides a simple API to render SVG images in your game.

Svg support is provided by the `flame_svg` external package, be sure to put it in your pubspec file
to use it.

To use it just import the `Svg` class from `'package:flame_svg/flame_svg.dart'`, and use the
following snippet to render it on the canvas:

```dart
Svg svgInstance = Svg('android.svg');

final position = Vector2(100, 100);
final size = Vector2(300, 300);

svgInstance.renderPosition(canvas, position, size);
```

or use the [SvgComponent]:

```dart
class MyGame extends FlameGame {
    Future<void> onLoad() async {
      final svgInstance = await Svg.load('android.svg');
      final size = Vector2.all(100);
      final svgComponent = SvgComponent.fromSvg(size, svgInstance);
      svgComponent.x = 100;

>       svgComponent.y = 100;

Reviewer 2: I would put the position in the constructor too (and add a trailing comma so that each argument in the constructor is on its own line).

      add(svgComponent);
    }
}
```

> More [here](https://docs.flame-engine.org/main/images.html#svg).

Reviewer 2: This link is not valid anymore since you moved that section

# Flame SVG

> Flame provides a simple API to render SVG images in your game.

Reviewer 2: ```suggestion
flame_svg provides a simple API for render SVG images in your game.
```

> flame_svg provides a simple API for render SVG images in your game.

Reviewer 2: ```suggestion
flame_svg provides a simple API for rendering SVG images in your game.
```

## Installation

> Svg support is provided by the `flame_svg` external package, be sure to put it in your pubspec file

Reviewer 2: ```suggestion
Svg support is provided by the `flame_svg` bridge package, be sure to put it in your pubspec file
```

to use it.

> If you want to know more about the installation visit [flame_svg on pubdev](https://pub.dev/packages/flame_svg/install).

Reviewer 2: ```suggestion
If you want to know more about the installation visit
[flame_svg on pubdev](https://pub.dev/packages/flame_svg/install).
```

## How to use flame_svg

To use it just import the `Svg` class from `'package:flame_svg/flame_svg.dart'`, and use the
following snippet to render it on the canvas:

```dart
Svg svgInstance = Svg('android.svg');

final position = Vector2(100, 100);
final size = Vector2(300, 300);

svgInstance.renderPosition(canvas, position, size);
```

> or use the [SvgComponent]:

Reviewer 2: ```suggestion
or use the `SvgComponent`:
```

> or use the `SvgComponent`:

Reviewer 2: ```suggestion
or use the `SvgComponent` and add it to the component tree:
```
